docs/javascript/v2/features/hls.mdx

```diff
@@ -145,6 +148,9 @@ function renderHLS({ hlsUrl }) {
 ```jsx
 import Hls from 'hls.js';
 
+const hlsState = useHMSStore(selectHLSState);
```

Comment:
Import this so that the snippet works for copy pasting?

Reply:
done